common/src/main/kotlin/entity/DiscordGuild.kt

```diff
@@ -81,82 +81,61 @@ data class DiscordGuild(
     val members: Optional<List<DiscordGuildMember>> = Optional.Missing(),
     val channels: Optional<List<DiscordChannel>> = Optional.Missing(),
     val threads: Optional<List<DiscordChannel>> = Optional.Missing(),
     val presences: Optional<List<DiscordPresenceUpdate>> = Optional.Missing(),
-    @SerialName("max_presences")
-    val maxPresences: OptionalInt? = OptionalInt.Missing,
-    @SerialName("max_members")
-    val maxMembers: OptionalInt = OptionalInt.Missing,
-    @SerialName("vanity_url_code")
-    val vanityUrlCode: String?,
+    @SerialName("max_presences") val maxPresences: OptionalInt? = OptionalInt.Missing,
+    @SerialName("max_members") val maxMembers: OptionalInt = OptionalInt.Missing,
+    @SerialName("vanity_url_code") val vanityUrlCode: String?,
     val description: String?,
     val banner: String?,
-    @SerialName("premium_tier")
-    val premiumTier: PremiumTier,
-    @SerialName("premium_subscription_count")
-    val premiumSubscriptionCount: OptionalInt = OptionalInt.Missing,
-    @SerialName("preferred_locale")
-    val preferredLocale: String,
-    @SerialName("public_updates_channel_id")
-    val publicUpdatesChannelId: Snowflake?,
-    @SerialName("max_video_channel_users")
-    val maxVideoChannelUsers: OptionalInt = OptionalInt.Missing,
-    @SerialName("approximate_member_count")
-    val approximateMemberCount: OptionalInt = OptionalInt.Missing,
-    @SerialName("approximate_presence_count")
-    val approximatePresenceCount: OptionalInt = OptionalInt.Missing,
-    @SerialName("welcome_screen")
-    val welcomeScreen: Optional<DiscordWelcomeScreen> = Optional.Missing(),
-    @SerialName("nsfw_level")
-    val nsfwLevel: NsfwLevel
+    @SerialName("premium_tier") val premiumTier: PremiumTier,
+    @SerialName("premium_subscription_count") val premiumSubscriptionCount: OptionalInt = OptionalInt.Missing,
+    @SerialName("preferred_locale") val preferredLocale: String,
+    @SerialName("public_updates_channel_id") val publicUpdatesChannelId: Snowflake?,
+    @SerialName("max_video_channel_users") val maxVideoChannelUsers: OptionalInt = OptionalInt.Missing,
+    @SerialName("approximate_member_count") val approximateMemberCount: OptionalInt = OptionalInt.Missing,
+    @SerialName("approximate_presence_count") val approximatePresenceCount: OptionalInt = OptionalInt.Missing,
+    @SerialName("welcome_screen") val welcomeScreen: Optional<DiscordWelcomeScreen> = Optional.Missing(),
+    @SerialName("nsfw_level") val nsfwLevel: NsfwLevel,
+    @SerialName("stage_instances")
+    val stageInstances: Optional<List<DiscordStageInstance>> = Optional.Missing(),
+    val stickers: Optional<List<DiscordMessageSticker>> = Optional.Missing(),
+    @SerialName("guild_scheduled_events")
+    val guildScheduledEvents: Optional<List<DiscordGuildScheduledEvent>> = Optional.Missing(),
+    @SerialName("premium_progress_bar_enabled")
+    val premiumProgressBarEnabled: OptionalBoolean = OptionalBoolean.Missing
```

Comment:
Is this really optional?

Reply:
Yes, tests were failing

Reply:
Because of Discord or the JSON test files?

Reply:
I'll give it another look; I defintely didn't update the json file